static/bidder-info/bidmatic.yaml

```diff
@@ -0,0 +1,18 @@
+endpoint: "http://adapter.bidmatic.io/pbs/ortb"
+maintainer:
+  email: "advertising@bidmatic.io"
```

Comment:
Prebid team has sent email to verify above maintainers email. Requesting to responding back on email thread

Reply:
received response from `advertising@bidmatic.io`